tests/manager/connect-error-001.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\Manager - no connection (command)
```

Comment:
I'd suggest naming the file `manager-executeCommand_error-003.phpt` (there is a gap between `002` and `004`, so we can fill it. And then "MongoDB\Driver\Manager::executeCommand() connection error" as a test name.

Down the line, we can make copies of this test for other execute methods on Manager and Server. The naming convention becomes much more practical in that case.